src/Java.Interop/Java.Interop/JavaArray.cs

```diff
@@ -12,6 +12,8 @@ namespace Java.Interop
 {
 	public abstract class JavaArray<T> : JavaObject, IList, IList<T>
 	{
+		internal const DynamicallyAccessedMemberTypes ConstructorsAndInterfaces = DynamicallyAccessedMemberTypes.PublicConstructors | DynamicallyAccessedMemberTypes.NonPublicConstructors | DynamicallyAccessedMemberTypes.Interfaces;
```

Comment:
I wonder if this const should instead go on `JavaObject`.  I *think* that would imply/require less "quaification", e.g. later in this PR there is a `JavaArray<T>.ConstructorsAndInterfaces` expression, which just feels weird when the value doesn't depend on `T` at all.